Modules/Effect/Fixture/Fixture/FixtureModule.cs

```diff
@@ -408,7 +408,7 @@ private void RenderCurve(Curve curve, FixtureFunction function, CancellationToke
 		private FixtureFunction GetFunction(string functionName)
 		{
 			// Return the first fixture function that matches the name
-			return _fixtureFunctions.Single(fn => fn.Name == functionName);
+			return _fixtureFunctions.First(fn => fn.Name == functionName);
 		}
```

Comment:
Are there any cases where First would not find a value and could throw an exception? The other changes used first or default and have checks so more of a sanity question here.

Reply:
I don't think so...but I have been spectacularly wrong before.
I can add some defensive code.

Reply:
No need to add defensive logic in unless it is warranted.  I will go ahead and merge.